lein-eftest/src/leiningen/eftest.clj

```diff
@@ -75,23 +75,23 @@
   ;;
   ;; Note, that we've duplicated the default reporter decision both here and in
   ;; `eftest.runner/run-tests`.
-  (let [reporter (or report eftest.report.progress/report)]
-    (cond-> (or options {})
-      (some? report-to-file)
-      (update :report eftest.runner/report-to-file report-to-file))))
+  `(let [reporter# (or ~report eftest.report.progress/report)]
+     (cond-> (or ~options {})
+       (some? ~report-to-file)
+       (update :report eftest.report/report-to-file ~report-to-file))))
```

Comment:
I'm not sure if there's a preferred way of doing this. The key here is that we defer referring to namespaces that only get required by the ultimately unquoted eval call down below.